deployment/server.js

```diff
@@ -97,11 +97,10 @@ io.on("connection", (socket) => {
 
     const room = roomManager.getRoom(descData.room);
     if (room.room.addNode) {
-      room.room.addNode(
-        descData.id,
-        MAX_CLIENTS_PER_HOST,
-        descData.selectedHost
-      );
+
+      if(room.room.addNode(descData.id, MAX_CLIENTS_PER_HOST, descData.selectedHost)) {
```

Comment:
I took a look at the code. We should probably rename room.room to room.networkTree

Reply:
The reason it's called room is because it doesn't necessarily have to be a room with a tree. If it's a room without distributed  streaming, it doesn't make sense to call it NetworkTree. We can change it down the line when we completely move to the distributed system after it's reliable enough